template/cordova/lib/MSBuildTools.js

```diff
@@ -38,6 +38,13 @@ MSBuildTools.prototype.buildProject = function(projFile, buildType, buildarch) {
     '/p:Configuration=' + buildType,
     '/p:Platform=' + buildarch];
 
+    if (otherConfigProperties) {
```

Comment:
This approach is not consistent with other ways we've done this in the code here. For packaging, we've leaned on generating a targets file every time we build. We should use that same mechanism here: https://github.com/apache/cordova-windows/blob/master/template/cordova/lib/prepare.js#L74

Also, one of the downsides of this approach is that IDE <-> command line interop is broken. It's desirable for the IDE build to have the same results as the command line build. By adding an additional property at command line this can't be guaranteed.